DUI3-DX/Sdk/Speckle.Connectors.Utils/Operations/ReceiveOperation.cs

```diff
@@ -41,12 +48,28 @@ public ReceiveOperation(IHostObjectBuilder hostObjectBuilder, ISyncToMainThread
 
     cancellationToken.ThrowIfCancellationRequested();
 
-    // 4 - Convert objects
-    return await _syncToMainThread
-      .RunOnThread(() =>
-      {
-        return _hostObjectBuilder.Build(commitObject, projectName, modelName, onOperationProgressed, cancellationToken);
-      })
-      .ConfigureAwait(false);
+    try
+    {
+      // 4 - Convert objects
+      var x = await _syncToMainThread
+        .RunOnThread(() =>
+        {
+          return _hostObjectBuilder.Build(
+            commitObject,
+            projectName,
+            modelName,
+            onOperationProgressed,
+            cancellationToken
+          );
+        })
+        .ConfigureAwait(false);
+
+      return x ?? new List<string>();
+    }
+    catch (Exception e)
```

Comment:
At best this should use our when !IsFatal() patten which @JR-Morgan can tell you about. Catching everything here I am not sure about, it's definitely sensible but we probably want a top level handler in the bridge and my question is, would they be different. We probably already had something bogus here. @JR-Morgan needs to chip in on this because it will bleed into reporting and exception handling.

Naturally the same applies to the receive. At this least the IsFatal should go here

Reply:
Happy to do that.  I was more concerned about the nullability because I was getting an exception swallowed and a null value back (that shouldn't be null)

Reply:
Also might tie into the logging work @JR-Morgan was looking at as this logger is separate from the one used while debugging it seems.

Reply:
Yeah we just have a policy of not swallowing everything, if it's Fatal we have to begin to warn the customer, hey really bad sh#t happened so you might want to not continue editing your critical path Revit construction file :D

Reply:
I did have some questions about why the return type was nullable from a lot of this, if that were possible or whether we should be saying it isn't but I'll let @JR-Morgan weigh-in on this.

Reply:
Just to clarify then, 
are we saying that sometimes, `_hostObjectBuilderBuild` is returning `null`?

If so, it shouldn't be! and I don't think we should be trying to handle it here. If Its marked as `nonnullable` then its not on the caller to double check this...